modules/core/src/main/scala/org/tessellation/infrastructure/trust/TrustDaemon.scala

```diff
@@ -26,8 +25,8 @@ object TrustDaemon {
         _ <- S.supervise(modelUpdate.foreverM).void
       } yield ()
 
-    private def calculatePredictedTrust(trust: Map[PeerId, TrustInfo]): Map[PeerId, Double] =
-      TrustModel.calculateTrust(trust, selfPeerId)
+    private def calculatePredictedTrust(trust: TrustMap): Map[PeerId, Double] =
+      TrustModel.calculateTrust(trust.trust, trust.peerLabels, selfPeerId)
```

Comment:
You could change the `calculateTrust` signature and pass the `TrustMap` directly if both vars inside `TrustMap` are needed. Not crucial, just a suggestion.

Reply:
Changed the signature of `calculateTrust`.